dbms/src/Flash/Coprocessor/InterpreterDAG.cpp

```diff
@@ -65,12 +66,24 @@ BlockInputStreams InterpreterDAG::executeQueryBlock(DAGQueryBlock & query_block)
         BlockInputStreams child_streams = executeQueryBlock(*child);
         input_streams_vec.push_back(child_streams);
     }
-    DAGQueryBlockInterpreter query_block_interpreter(
-        context,
-        input_streams_vec,
-        query_block,
-        max_streams);
-    return query_block_interpreter.execute();
+    if (context.getSettingsRef().enable_planner && Planner::isSupported(query_block))
+    {
+        Planner planner(
```

Comment:
planner still need `query_block` as its input?

Reply:
After all operators are converted to physical plan, we can get out of the query block.

Reply:
In this way, the operators can be converted into physical plan one by one